showcase/app/templates/layouts/grid.hbs

```diff
@@ -44,16 +55,56 @@
       </Shw::Outliner>
     </SG.Item>
 
-    <SG.Item @label="30em min width columns">
+    <SG.Item @label="33.33% min width columns w/ 2 items">
       <Shw::Outliner>
-        <Hds::Layout::Grid @gap="24" @columnMinWidth="30em">
+        <Hds::Layout::Grid @gap="24" @columnMinWidth="33.33%">
+          <Shw::Placeholder @text="#1" @height="40" />
+          <Shw::Placeholder @text="#2" @height="40" />
+        </Hds::Layout::Grid>
+      </Shw::Outliner>
+    </SG.Item>
+
+    <SG.Item @label="33.33% min width columns w/ 1 item">
+      <Shw::Outliner>
+        <Hds::Layout::Grid @gap="24" @columnMinWidth="33.33%">
+          <Shw::Placeholder @text="#1" @height="40" />
+        </Hds::Layout::Grid>
+      </Shw::Outliner>
+    </SG.Item>
+  </Shw::Grid>
+
+  <Shw::Divider @level={{2}} />
+
+  <Shw::Text::H2>Column width</Shw::Text::H2>
```

Comment:
[suggestion] I would probably do something like this:

- Column width management (h2)
    - ColumnWidth (h3)
    - ColumnMinWidth (h3)